zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/AbstractSchema.scala

```diff
@@ -199,26 +199,26 @@ abstract class Schema(typeName: String,
 
   def getTypeName: String = typeName
   def getTimeField: TimeField
```

Comment:
if that is the case, I think we should remove the `hasTimeField` method, and let `getTimeField` returns an `Option[Timefield]`?